server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -126,6 +126,8 @@ private FormTag buildProgramForm(
             .setFieldName("localizedDisplayDescription")
             .setLabelText("Describe this program for the public")
             .setRequired(true)
+            .setMarkdownSupported(true)
```

Comment:
(question) Should these have the setMarkdownLinkText as well?

Reply:
No, Sijia's designs only wanted to link out the docs from the question creation/edit page because we have some caveats about how to use markdown on those pages.